src/zsclient.cpp

```diff
@@ -898,7 +898,7 @@ namespace zsync2 {
 
             // check whether file exists at all, because if not, a full download is required
             if (!isfile(pathToLocalFile)) {
-                issueStatusMessage("Cannot find file " + pathToLocalFile + ", triggering full download");
+                issueStatusMessage("Cannot find file " + pathToLocalFile);
```

Comment:
That's more misleading than before. I already explained what the message means. Now it's more than ambiguous...

Reply:
It is _not_ "triggering a full download" if further down below there is a file supplied with `-i`.

Reply:
True. But what's the file and why does it look for it at all?

Reply:
Remove the message altogether?

Reply:
Why not state that this filename has been taken from the `.zsync` file, but we couldn't find it?

Reply:
"Skipping file (...) as it is not present on the local system"?

Reply:
That also doesn't tell "why" it's looking for that file. Please add some comment with regards to this.

Reply:
Skipping file (...) suggested by (...).zsync  because it is not found locally at (...)

Reply:
I guess `.zsync file` is sufficient. Not sure what the `at` part is for. The message doesn't tell what it wants to use the file for. It's like an implicit `-i`, isn't it?

Reply:
How about just mentioning the file if it could be found? Something like "Found file ... suggested by .zsync file locally, trying to use data from it"?

Reply:
Great!